### ISSUE TITLE
questions about step3 training on single image

### ISSUE THREAD

**Issue by ianyhx** [MathiasGruber/PConv-Keras#15]
@MathiasGruber Thank you very much for your excellent code! I have come up against some problems when running your code in Step 3, about training UNet on single image. I've downloaded your pretrained VGG weights and put it in model. However, I found the loss decrease very slow, and the predicted image seems not right.

Epoch 1/10
2000/2000 [==============================] - 1239s 619ms/step - loss: 5.1134 - PSNR: 11.9789
![1](https://user-images.githubusercontent.com/13147099/53927287-51fe5900-40c1-11e9-9626-6497211471f1.png)

Epoch 2/10
2000/2000 [==============================] - 1226s 613ms/step - loss: 2.8002 - PSNR: 16.5682
![2](https://user-images.githubusercontent.com/13147099/53927294-56c30d00-40c1-11e9-9113-482750456fe1.png)

Epoch 3/10
2000/2000 [==============================] - 1231s 616ms/step - loss: 2.2371 - PSNR: 19.1468
![3](https://user-images.githubusercontent.com/13147099/53927486-04362080-40c2-11e9-9bdf-40486e87ead8.png)

Is there some wrong with my implementation?

**Comment by SunnyPann** [comment]
Are you training on your own training set?Where can I download the training model?thank you

**Comment by zjlinkin** [comment]
I have had the same situation, have you find out why or solved this problem?
